src/v1/spec.yaml

```diff
@@ -1045,7 +1045,7 @@ paths:
     get:
       tags:
         - Cast
-      summary: DEPRECATED - Fetch casts for a given user
+      summary: DEPRECATED - Fetch casts for a user
```

Comment:
This one is deprecated so matters less but maybe something like `Casts by user`?